### PR TITLE
Update dependencies and remove unnecessary code

### PR DESCRIPTION
**Pull request by friederbluemle**
A bunch of project updates and cleanups. Please check individual commits for details.

Update for the latest stable version of Android Studio 3.2.0
Also fixes all Lint warnings and whitespace errors.

<sub><sup>Once approved, please use a *normal* GitHub merge (i.e. **NO rebase/squash** merge) to integrate the commit(s) from the branch of this pull request. The changes are broken up into meaningful, [atomic commits](https://www.freshconsulting.com/atomic-commits/), and my branch should already be up-to-date with the latest target branch. If it isn't, or if you'd like me to combine something, please let me know, and I will make the necessary updates as soon as possible. Thank you!</sup></sub>

### CODE REVIEW

**Comment by mrmike** [comment]
Hi @friederbluemle 

Thank you for your contribution. I'll review pull request soon.

Thanks

**Comment by friederbluemle** [comment]
Thanks @mrmike 👍

File: build.gradle [hunk 1]
```diff
@@ -1,26 +1,22 @@
...
         google()
+        jcenter()
     }
 }
 
 task clean(type: Delete) {
     delete rootProject.buildDir
 }
-
-task wrapper(type: Wrapper) {
```
Comment (mrmike) [inline build.gradle]: Please keep this wrapper task and update `gradleVersion` here to the latest one.

Comment (mrmike) [reply build.gradle]: Great I didn't now that Gradle support that. Good to know :)

**Pull request merged**